flake.nix

```diff
@@ -14,14 +14,15 @@
         openssl_1_1_1 = import ./nix/openssl_1_1_1.nix { pkgs = pkgs; };
         openssl_3_0 = import ./nix/openssl_3_0.nix { pkgs = pkgs; };
         corretto-8 = import nix/amazon-corretto-8.nix { pkgs = pkgs; };
-        gnutls-3-7 = import nix/gnutls.nix { pkgs = pkgs; };
         common_packages = [
           # Integration Deps
-          # The openssl and gnutls imports above are imnplicitly included in mkShells.
+          # The openssls import above is imnplicitly included in mkShells.
```

Comment:
nit: There's two OpenSSL imports above, but it's just 1.1.1 that's used implicitly in the mkShell, right? Or am I misunderstanding this comment? Also typo fix:
```suggestion
          # The openssl 1.1.1 import above is implicitly included in mkShells.
```

Reply:
Actually, second thought, I rewrote this whole comment.  The _derivation_  for 1.1.1 gets built, but it's only useful because we're manually inserting it into the PATH.